browser-test/test.endtoend.ts

```diff
@@ -60,9 +60,9 @@ describe('Run tests against gRPC server', () => {
     };
     const MAX_RETRIES = 20;
     const TIMEOUT = 1000;
-
+    let retryCount = 0;
     // Making sure server is up before starting tests.
-    for (let retryCount = 0; retryCount < MAX_RETRIES; ++retryCount) {
+    for (retryCount; retryCount < MAX_RETRIES; ++retryCount) {
```

Comment:
I don't remember, but do you actually need to put the variable here? Can it just be `for(; retryCount < MAX_RETRIES...`?